meson.build

```diff
@@ -2930,6 +2930,21 @@ if conf.get('ENABLE_SYSUSERS') == 1
                 install_rpath : rootlibexecdir,
                 install : true,
                 install_dir : rootbindir)
+
+        if get_option('standalone-binaries')
+                public_programs += executable(
+                        'systemd-sysusers.standalone',
+                        'src/sysusers/sysusers.c',
+                        include_directories : includes,
+                        link_with : [libshared_static,
+                                     libbasic,
+                                     libbasic_gcrypt,
+                                     libsystemd_static,
+                                     libjournal_client],
+                        install_rpath : rootlibexecdir,
+                        install : true,
```

Comment:
It'd be better to move the conditional here, i.e. only build and install the binary if configured, but allow building on demand always. Something like
```python
           build_by_default: have_standalone_binaries,
           install : have_standalone_binaries,
```

Reply:
Good call!

I also noticed that `install_rpath` is not needed here, since it was actually needed for the `systemd-shared` solib, which is no longer present.

Tested the `have_standalone_binaries` change and it works as expected for me.